src/helpers.js

```diff
@@ -7,19 +7,19 @@ const eventPath = (event) => {
 };
 
 const fileMissing = (event) => {
-  return !/\.(jpg|tif|gif|png|webp|json)$/.test(event.path);
+  return !/\.(jpe?g|tiff?|gif|png|webp|json)$/.test(event.path);
 };
 
 const getUri = (event) => {
   const scheme = event.headers['X-Forwarded-Proto'] || 'http';
-  const host = event.headers['X-Forwarded-Host'] || event.headers.Host;
+  const host = process.env.forceHost || event.headers['X-Forwarded-Host'] || event.headers.Host;
```

Comment:
I suspect this would help with custom domains? I wonder if documentation about how to host this at a custom domain would be useful?

Reply:
Custom domains are precisely the reason this bit exists. But you're right that I should add some documentation for the SAM/CloudFormation parameters that are not currently documented.